package.json

```diff
@@ -26,7 +26,9 @@
     "metatests": "./bin/cli.js"
   },
   "scripts": {
-    "test": "npm run lint && ./bin/cli.js test",
+    "test": "npm run lint && ./bin/cli.js test --exclude test/fixtures",
+    "unittests": "node test/unittests",
```

Comment:
What's this command for?

Reply:
To be able to set special command in `stryker.conf.js` for running tests.

Reply:
Oh, missed it. Then maybe `"test:unit"`?